frontend/src/Pages/SearchPage.tsx

```diff
@@ -64,62 +64,61 @@ function SearchResult(props: {
     </>;
 }
 
+let latestSearchTerm = '';
```

Comment:
This is not an idiomatic approach, as it would not work if we use two instances of the SearchPage component (probably won't happen in our case, but still).

The correct approach would be to utilize [`useRef`](https://beta.reactjs.org/reference/react/useRef) hook, or our wrapper [`usePrevious`](https://github.com/spaceshelter/orbitar/blob/0d0deb35837330217a4e3c523332a17d47b5190f/frontend/src/API/use/usePrevious.ts#L7)